<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>742</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>742</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#585492;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="searchEdit_2">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #FB8A2D;
border: 1px solid #253261;</string>
          </property>
          <property name="placeholderText">
           <string>Название игры</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableWidget {background-color: #253261; color: #FB8A2D; font-weight: bold}
QHeaderView::section { background: #253261; color: #FB8A2D; font-weight: bold };





</string>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearBtn">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #FB8A2D;</string>
          </property>
          <property name="text">
           <string>Очистить всё</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="changeBox">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(251, 138, 45);</string>
            </property>
            <property name="editable">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>CSV</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>EXCEL</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>TXT</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="unloadBtn">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #FB8A2D;</string>
            </property>
            <property name="text">
             <string>Выгрузить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
